redis_sentinel/datadog_checks/redis_sentinel/redis_sentinel.py

```diff
@@ -13,9 +13,33 @@ def __init__(self, name, init_config, agentConfig, instances=None):
         AgentCheck.__init__(self, name, init_config, agentConfig, instances)
         self._masters = defaultdict(lambda: "")
 
+    def _load_config(self, instance):
+        host = instance.get('sentinel_host')
+        port = instance.get('sentinel_port')
+
+        if not host or not port:
+            raise ConfigurationError('Configuration error, please fix redis_sentinel configuration')
+
+        try:
+            if float(port) != int(port):
```

Comment:
```suggestion
            if isinstance(port, int):
```

Let's directly check if the port is an int so we don't need the try/except block with duplicate error message.

Reply:
Valid, thanks for this and implemented.